recipes/pyexodus/meta.yaml

```diff
@@ -0,0 +1,82 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "pyexodus" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "6ff3be9bbe83af0e0443f5b0a9e23f5fe3f73f117b02c7d5a20fcbd41c91c000" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub remove the line above
```

Comment:
Can you please cleanup the recipe a little bit and remove the comments.
Thanks!

Reply:
Sure thing. Would you prefer to have all comments removed or only those that don't apply to this particular recipe? I actually think they are kind of helpful if one only occasionally looks at them.

Reply:
All that do not really apply to this specific recipe. Its good to learn how things should be but in an official version they should imho be removed.